data/json/recipes/armor/boots.json

```diff
@@ -16,6 +16,23 @@
     ]
   },
   {
+    "result": "boots_chitin",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
+    "skill_used": "tailor",
+    "difficulty": 6,
```

Comment:
5 is light survivor armor, 6 is medium, 7 is heavy. 6 is when tailoring fully shines.
We need good recipes for level 4, which is rather empty (just leg stuff).